src/com/oltpbenchmark/util/PlacementBlock.java

```diff
@@ -0,0 +1,52 @@
+package com.oltpbenchmark.util;
+
+/*
+ * For GeoPartitioned clusters, the configuration specifying how the
+ * tables should be placed is a list of cloud.region.zone.min_number_of_replicas
+ * blocks. 'PlacementBlock' class represents one such c.r.z.m block.
+ */
+public class PlacementBlock {
+    private String cloud;
+    private String region;
+    private String zone;
+    private int min_num_replicas;
+
+    public PlacementBlock(String cloud, String region, String zone, int min_num_replicas) {
+        this.cloud = cloud;
+        this.region = region;
+        this.zone = zone;
+        this.min_num_replicas = min_num_replicas;
+    }
+
+    public String getCloud() {
+        return cloud;
+    }
+
+    public void setCloud(String cloud) {
```

Comment:
it looks like setters are not used. Please remove them and make all member fields `final`

Reply:
For some reason I thought the Gson library needed them, I was mistaken. Removed these